tests/test_offchain.py

```diff
@@ -40,7 +42,7 @@ def test_prepare_create_transaction_default(alice_pubkey):
 
 
 @mark.parametrize('asset', (
-    None, {}, {'data': {'msg': 'Hello BigchainDB!'}},
+    None, {'data': None}, {'data': {'msg': 'Hello BigchainDB!'}},
```

Comment:
Note that any `asset` dict being passed into `prepare_transaction()` needs to be either of the following formats:

* `CREATE`:
```json
{
    "data": ...
}
```

* `TRANSFER`:
```json
{
    "id": ...
}
```